includes/class-give-license-handler.php

```diff
@@ -516,6 +521,10 @@ private function __single_subscription_check() {
 				return;
 			}
 
+			// Bailout.
```

Comment:
@DevinWalker I am verifying API response. In some case i will get limit access API response